spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -1005,6 +1011,8 @@ class FigureCanvas(QFrame):
     """
     A basic widget on which can be painted a custom png, jpg, or svg image.
     """
+    sig_clear_fig_request = Signal()
```

Comment:
I think signals are supposed to be named more like

`sig_clear_fig_requested` ?